infrastructure/kctf-samples/apache-php/challenge/config/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
 configMapGenerator:
 - name: apache-php-config
   files:
-  - apache-nsjail-php.conf
+  - apache2-nsjail-php.conf
```

Comment:
this is interesting, I changed the name of the file at some point, but the deployment still worked for some reason.

@stephenR

Reply:
Hmm, that should be applied via this line in the Makefile:
kubectl apply -k k8s/deployment
This should always run and kubectl should pick up the differences.
You mean you made a change that should've been broken but it still worked?

Reply:
yea, this change should have broken everything, but somehow didn't.